tests/PHPStan/Analyser/data/param-out.php

```diff
@@ -485,3 +485,20 @@ function ($s, $t): void {
 	assertType('string', $s);
 	assertType('string', $t);
 };
+
+function fooMatch(string $input): void {
```

Comment:
expectations have been harmonized across php versions, therefore the 2 variants above removed and the tests moved with same expectations in this common file